.travis.yml

```diff
@@ -14,7 +14,7 @@ branches:
 install:
   - pip install --upgrade numpy cython
   - pip install -e .
-
+  - pip install pytest pytest-cov
 
 script:
-  - true
+  - pytest --cov=./
```

Comment:
`pytest --cov mdbenchmark -v --pep8 mdbenchmark`

this means you also need to install `pytest-pep8`